server/routes/appointments/appointmentsController.ts

```diff
@@ -471,7 +486,7 @@ export default class AppointmentsController {
         let basePath
         let redirectPath
         if (draftBookingId) {
-          const fetchResult = await this.fetchDraftAppointmentOrRenderMessage(req, res)
+          const fetchResult = await this.fetchDraftAppointmentOrRenderMessage(req, res, 'service-provider')
```

Comment:
I am seeing you are passing the `userType` variable many places, is it possible to identity from the  request?

Reply:
> I am seeing you are passing the `userType` variable in many places, is it possible to identity from the request?. This comment holds good for all the places where you are passing this.

I don't think it's possible, the 'userType' is already being passed through to some controllers like this and I applied the same approach.

Reply:
Ok no worries then.